TeamHangman/TeamHangman/Game.cpp

```diff
@@ -75,6 +80,44 @@ void Game::WordVectorInit()
 	}
 }
 
+void Game::LettersVectorInit()
+{
+	std::string word = GetRandomLine();
+	std::cout << word << std::endl;
+	std::string letter;
+	float letterSpacing;
+	float firstLetterPos;
+	float lastLetterPos;
+	if (word.size() <= 10)
+	{
+		firstLetterPos = 0.3f;
+		lastLetterPos = 0.7f - firstLetterPos;
+		letterSpacing = window->getSize().x * lastLetterPos / word.size();
+	}
+	else if (word.size() >= 30)
+	{
+		firstLetterPos = 0.05f;
+		lastLetterPos = 0.95f - firstLetterPos;
+		letterSpacing = window->getSize().x * lastLetterPos / word.size();
+	}
+	else
+	{
+		firstLetterPos = 0.1f;
+		lastLetterPos = 0.9f - firstLetterPos;
+		letterSpacing = window->getSize().x * lastLetterPos / word.size();
```

Comment:
We could calculate letterSpacing just once, below these three if, else if, else statements.

Reply:
true dat